pkg/util/pki/subject_test.go

```diff
@@ -139,7 +140,7 @@ func TestRoundTripRDNSequence(t *testing.T) {
 		},
 		{
 			[]pkix.AttributeTypeAndValue{
-				{Type: OIDConstants.CommonName, Value: "fo\x00o-long.com"},
+				{Type: asn1.ObjectIdentifier{0, 5, 80, 99, 58962185}, Value: "fo\x00o-long.com"},
```

Comment:
Why have you replaced that existing `OIDConstants.CommonName` test case?
Is it because CommonName was already tested in line 130 above?

Suggest adding a comment above this test case explaining that it is the `test case for literal OID feature`

Reply:
I added names to the testcases.